tests/Fixtures/App/DataFixtures/Phpcr/LoadContentData.php

```diff
@@ -9,7 +9,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Cmf\Bundle\ContentBundle\Tests\Resources\DataFixtures\Phpcr;
+namespace Symfony\Cmf\Bundle\ContentBundle\Tests\Fixttures\App\DataFixtures\Phpcr;
```

Comment:
typo: Fixtures with just one `t`

Reply:
uhhh nice ... this is a real maxx :-)

Reply:
but here we see, that the fixture isn't used from this bundle

Reply:
are you sure? is it not just that the fixtures are loaded by folder and the loader does not care what namespace/class name is used and just executes the fixture?

if the file is not used, we should delete it instead.